### PR TITLE
detox: add page

### PR DESCRIPTION
**Pull request by slicer69**
Detox is a program which removes spaces, duplicate underlines and other unwanted characters from filenames.

### CODE REVIEW

File: pages/common/detox.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# detox
+
+> Renames files to make them easier to work with. It removes spaces
```
Comment (agnivade) [inline pages/common/detox.md]: Move the "It removes spaces .." to the next line.

File: pages/common/detox.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# detox
+
+> Renames files to make them easier to work with. It removes spaces 
+> and other such annoyances like duplicate underline characters. 
+
+- Remove spaces and other undesirable characters from a file's name:
+
+`detox {{file}}`
+
+- Show how detox would rename all of the files in a directory tree:
```
Comment (agnivade) [inline pages/common/detox.md]: `-n` is for dry-run right ? How about - `Perform a dry-ru[n] of detox recursively on all files in a given directory`

Comment (pxgamer) [reply pages/common/detox.md]: Could use the full option of `--dry-run`?

Comment (agnivade) [reply pages/common/detox.md]: Well, sure. That's why I highlighted the `[n]` in dry-run so that it's understood. But I am okay with both approaches.

**Comment by sbrl** [comment]
Would this be a squash & merge, or an interactive rebase @agnivade?

**Comment by agnivade** [comment]
The commit message for the 2nd commit is not very well worded. So, if you are up for it, you can update the commit message for that and do a "rebase and merge". Else, just squash.

**Comment by sbrl** [comment]
I've merged this in #2047 as I made a mistake! So even though I'm closing this PR, @slicer69 - I've still merged your changes. Apparently I still have a lot to learn about rebasing.... :P

**Comment by slicer69** [comment]
@sbrl, thank you for merging my detox page.

**Pull request closed without merging**